src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,122 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private Node<K, V> nullKeyNode;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+    }
+
+    private static class Node<K, V> {
```

Comment:
Private classes and methods should be located after public ones in your class. Move the Node class after the public methods.

Reply:
Fix comment